examples/deployment-manager/autogen/singlevm/README.md

```diff
@@ -1,82 +1,72 @@
+This example uses the `mpdev` tool to generate a Deployment Manager template for a wordpress Google Compute Engine virtual machine (VM).
+
+To generate a Deployment Manager template for your VM product, 
+see the Google Cloud Marketplace documentation for [Creating your deployment package](https://cloud.google.com/marketplace/docs/partners/vm/create-deployment-package).
+
 ## Prerequisites
 
-This example package generates a Deployment Manager template for a wordpress GCE
-VM. Your GCP project must contain a wordpress VM image to use in the Deployment
-Manager template. The following command copies the wordpress VM image from the
+Your Google Cloud project must contain a wordpress VM image to use in the Deployment
+Manager template. Run the following command to copy the wordpress VM image from the
 [click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy)
-to your GCP project.
+to your project:
```

Comment:
> Run the following command to copy the wordpress VM image from the
> [click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy)
> to your project:

To copy the Wordpress VM image from the [click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy) to your project, run the following command: